server/task.js

```diff
@@ -0,0 +1,11 @@
+var db = require("./connection");
+
+var task = db.model('mytasks', {
+    title: {type: String, required: true},
+    dueDate: {type: String, required: true},
+    address: {type: String, required: true},
+    lat: {type: Number, required:true },
+    lng: {type:Number, required:true}
```

Comment:
Use two spaces for indentation

Reply:
Really? I have always used tab for indentation. I will update that and pretty much everywhere too.

Reply:
Tabs are fine too, but we should choose one way to do it and stick with it. Here you are using 4 spaces, not a tab, in any case. Maybe your editor is set to do 4 spaces by default?